config/icinga/Plugins_Bundle.conf

```diff
@@ -3955,7 +3963,7 @@ object CheckCommand "Invoke-IcingaCheckICMP" {
 
     arguments += {
         "-WarningPl" = {
-            description = "Threshold on which the plugin will return 'WARNING' for possible packet loss in %"
+            description = "Threshold on which the plugin will return 'WARNING' for the response time in ms"

```

Comment:
Is that correct? Either the option name is rather misleading or this might be a type